src/test/java/study/racingcar/HistoryTest.java

```diff
@@ -0,0 +1,28 @@
+package study.racingcar;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class HistoryTest {
+
+
+    @DisplayName("우승자를 리스트로 반환")
+    @Test
+    void winners() {
+        Car carA = new Car("a");
+        Car carB = new Car("b");
+
+        Map<Car, String> map = Map.of(carA, "----", carB, "-");
+
+        History history = new History(new LinkedList<>(List.of(map)));
+        List<String> winners = history.winners();
+        assertThat(winners.size()).isEqualTo(1);
+        assertThat(winners.get(0)).isEqualTo(carA.name());
+        assertThat(winners).doesNotContain(carB.name());
```

Comment:
winners 메서드가 이름들을 반환하고 있으니 우승자 이름을 직접 비교해도 좋을 것 같아요
```suggestion
        assertThat(winners).containsExactly("a");
```